spaceship/__init__.py

```diff
@@ -1,5 +1,23 @@
 from flask import Flask
+from flask_assets import Environment, Bundle
+
 app = Flask(__name__)
+assets = Environment(app)
+
+# If you need the HTML to include each JS file via its own <script> tag,
+# uncomment this line:
+# assets.debug = True
```

Comment:
is there a configuration option to the app that we can set in `config.py` to accomplish this? i wonder if we should default to `True` in dev but set it to `False` in prod?